helm/values.yaml

```diff
@@ -92,10 +92,6 @@ serviceSelectorLabels:
 entityServiceConfig:
   name: entity-service-config
   publishChangeEvents: false
-  dataStoreType: "mongo"
```

Comment:
So previously the values defaulted it to mongo - can we provide that equivalent config in the new format?

Reply:
I guess not really needed since the application conf already defaults it in the same way - but might be worth putting something in here just to show where the values would go if they need to be configured like we're doing lines 36-44

Reply:
Done. Added below.